daemon/daemon_unix.go

```diff
@@ -159,6 +159,21 @@ func getBlkioWeightDevices(config containertypes.Resources) ([]specs.WeightDevic
 	return blkioWeightDevices, nil
 }
 
+func parseAndSetSecurityOpt(daemon *Daemon, container *container.Container, config *containertypes.HostConfig) error {
```

Comment:
Seems it is a bit hard to pass daemon.configStore.NoNewPrivileges, because it only exist in unix, but our parseAndSetSecurityOpt is across platform...